proto/tendermint/budget/v1beta1/query.proto

```diff
@@ -0,0 +1,127 @@
+syntax = "proto3";
+
+package cosmos.budget.v1beta1;
+
+import "tendermint/budget/v1beta1/budget.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "cosmos_proto/cosmos.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+option go_package = "github.com/crescent-network/crescent/v5/x/budget/types";
+
+// Query defines the gRPC query service for the budget module.
+service Query {
+  // Params returns parameters of the budget module.
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get                                           = "/cosmos/budget/v1beta1/params";
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns all parameters of the budget module.";
+  external_docs: {
+  url:
+    "https://github.com/tendermint/budget/blob/main/x/budget/spec/06_params.md";
```

Comment:
We should update this docs URL.

Reply:
It's a branch that hasn't been released yet, so I'll just erase it because it's ambiguous to link it and maintenance will be cumbersome